source/sass/buyers-guide/views/homepage.scss

```
@@ -0,0 +1,57 @@
+#pni-home {
+  .page-header {
+    height: 400px;
+    background-image: url(/_images/buyers-guide/home-banner.jpg);
+    background-size: auto 400px;
+    background-repeat: no-repeat;
+    background-position: center center;
+  }
+
+  .product-listing {
+    margin-top: -88px;
+    padding-bottom: 88px;
+  }
+
+  .product-box {
+    width: calc(33%);
+    box-sizing: content-box;
+    margin: 1px;
+    overflow: hidden;
+  }
+
+  .creepiness-slider {
+    z-index: 0;
+    top: 40px;
+    padding-top: 20px;
+
```

Let's remove the `sticky-top` class and just add the following:
```suggestion
position: sticky;
position: -webkit-sticky;

```